Realm/RLMSchema.mm

```diff
@@ -126,12 +126,20 @@ - (instancetype)init {
 
 - (NSArray *)objectSchema {
     if (!_objectSchema) {
+        if (self == s_sharedSchema) {
+            // Ensure the schema is fully initialized
+            [RLMSchema sharedSchema];
```

Comment:
The only way for this code to do anything is if `objectSchema` is called on the shared schema obtained via `partialSharedSchema`. If that's something that can never actually happen, then it's dead code.